src/CardCloneDialog.vue

```diff
@@ -0,0 +1,120 @@
+<!--
+  - SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+<template>
+	<NcModal v-if="modalShow" :title="t('deck', 'Clone card')" @close="modalShow = false">
```

Comment:
Would it actually make sense to to the following?

- Call the action in the card menu "Copy/clone card"
- Merge both modals into one and just have two buttons "Copy" and "Move"

Reply:
Also throws a console warning:

```
[Vue warn]: [NcModal] You need either set the name or set a `labelId` for accessibility.

found in

---> <NcModal>
       <CardCloneDialog> at src/CardCloneDialog.vue
         <NcContent>
           <App> at src/App.vue
             <Root>
```